client/extensions/woocommerce/state/sites/settings/products/test/actions.js

```diff
@@ -9,15 +9,74 @@ import { spy } from 'sinon';
 /**
  * Internal dependencies
  */
-import { fetchSettingsProducts } from '../actions';
+import { fetchSettingsProducts, updateSettingsProducts } from '../actions';
 import useNock from 'test/helpers/use-nock';
 import {
+	WOOCOMMERCE_SETTINGS_PRODUCTS_UPDATE_REQUEST,
+	WOOCOMMERCE_SETTINGS_PRODUCTS_UPDATE_REQUEST_SUCCESS,
 	WOOCOMMERCE_SETTINGS_PRODUCTS_REQUEST,
 	WOOCOMMERCE_SETTINGS_PRODUCTS_REQUEST_SUCCESS,
 } from 'woocommerce/state/action-types';
 import { LOADING } from 'woocommerce/state/constants';
 
 describe( 'actions', () => {
+	describe( '#updateSettingsProducts()', () => {
```

Comment:
Since update settings has that array-ification business, please consider adding a test that passes an object instead of an array of objects.